### ISSUE TITLE
Broken contact details for EGL_EXT_surface_SMPTE2086_metadata and EGL_EXT_surface_CTA861_3_metadata

### ISSUE THREAD

**Issue by amyspark** [KhronosGroup/EGL-Registry#183]
Hi,

I've not found any other place to report this, hope this is the correct one. I've been trying to get ahold of the persons of contact for the following extensions:

https://github.com/KhronosGroup/EGL-Registry/blob/6550e9b12e0b4f11182bda30ae2a8dd56d0b02c8/extensions/EXT/EGL_EXT_surface_SMPTE2086_metadata.txt#L1-L11

https://github.com/KhronosGroup/EGL-Registry/blob/6550e9b12e0b4f11182bda30ae2a8dd56d0b02c8/extensions/EXT/EGL_EXT_surface_CTA861_3_metadata.txt#L1-L11

but both addresses bounce back. My contacts have also been unable to supply me with alternative points of contact.

Would it be possible to update the extensions' RFCs with up-to-date details?

**Comment by oddhack** [comment]
@jackohound I'm thinking we should probably update the extensions to recommend people ask questions here first - some of those email contacts are more than a decade old and many are probably no longer valid. For Vulkan we ask for github handles for extension contacts, rather than email.

**Comment by stonesthrow** [comment]
Is there an issue, besides stale email, that needs to be addressed?
I believe these to be current
@courtney-g 
@cubanismo 
@dgkoch 
@jackohound

**Comment by amyspark** [comment]
@stonesthrow No, I was attempting to get in touch with them to get information as regards the two extensions. I ran into the following ANGLE feature request:

https://bugs.chromium.org/p/angleproject/issues/detail?id=7826

And I was unable to find any docs, production use cases, etc. to see if I could write support for them as part of the HDR patchset I'm preparing.

**Comment by stonesthrow** [comment]
If you are not already working with Chris Glover, then that is a resource from Google. And he's been working colorspaces.
https://github.com/KhronosGroup/EGL-Registry/pull/177
@cdglove

**Comment by stonesthrow** [comment]
@amyspark lets close this if no further action

**Comment by amyspark** [comment]
@stonesthrow if I could get directed to whomever is handling those two extensions at present, that'd be great. My email is on my profile.

**Comment by stonesthrow** [comment]
These jira issues are to report a problem or a request for feature. So you can contact the IDs I gave for details or insight, or you can state a problem/request. Over time the original authors move on to other things. We don't really reassign to someone else.
If you state a request for change, then the contributors can make comment on that.
